PyInstaller/utils/hooks/__init__.py

```diff
@@ -1369,6 +1369,73 @@ def include_or_exclude_file(
     return True
 
 
+def collect_delvewheel_libs_directory(package_name, libdir_name=None, datas=None, binaries=None):
+    """
+    Collect data files and binaries from the .libs directory of a delvewheel-enabled python wheel. Such wheels ship
+    their shared libraries in a .libs directory that is located next to the package directory, and therefore falls
+    outside the purview of the collect_dynamic_libs() utility function.
+
+    Args:
+        package_name:
+            Name of the package (e.g., scipy).
+        libdir_name:
+            Optional name of the .libs directory (e.g., scipy.libs). If not provided, ".libs" is added to
+            ``package_name``.
+        datas:
+            Optional list of datas to which collected data file entries are added. The combined result is retuned
+            as part of the output tuple.
+        binaries:
+            Optional list of binaries to which collected binaries entries are added. The combined result is retuned
+            as part of the output tuple.
+
+    Returns:
+        tuple: A ``(datas, binaries)`` pair that should be assigned to the ``datas`` and ``binaries``, respectively.
+
+    Examples:
+        Collect the ``scipy.libs`` delvewheel directory belonging to the Windows ``scipy`` wheel::
+
+            datas, binaries = collect_delvewheel_libs_directory("scipy")
+
+        When the collected entries should be added to existing ``datas`` and ``binaries`` listst, the following form
+        can be used to avoid using intermediate temporary variables and merging those into existing lists::
+
+            datas, binaries = collect_delvewheel_libs_directory("scipy", datas=datas, binaries=binaries)
+
+    .. versionadded:: 5.6
+    """
+
+    datas = datas or []
+    binaries = binaries or []
+
+    if libdir_name is None:
+        libdir_name = package_name + '.libs'
+
+    # At the time being, delvewheel is applicatble only to windows wheels
```

Comment:
```suggestion
    # At the time being, delvewheel is applicable only to windows wheels
```
I'd be surprised if that ever changes. macOS has delocate and Linux has auditwheel.

Reply:
Good point. I've also removed "At the time being" part to reflect that.